### PR TITLE
Add fedmenu.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: fedoracommunity/templates/footer.mak [hunk 1]
```diff
@@ -16,5 +16,21 @@
             </ul>
           </div>
         </div>
+
+        % if 'fedmenu.url' in config:
+        <script src="${config['fedmenu.url']}/js/fedora-libravatar.js"></script>
```
Comment (pypingou) [inline fedoracommunity/templates/footer.mak]: Is this used in this case?

**Comment by pypingou** [comment]
One question, otherwise :+1:

File: fedoracommunity/templates/footer.mak (hunk 1 above)
Comment (ralphbean) [inline fedoracommunity/templates/footer.mak]: Nope!  Removed in e3a95d6

Comment (ralphbean) [inline fedoracommunity/templates/footer.mak]: Thanks for the catch :)

**Pull request merged**